ecs_deploy.sh

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env bash
+# source: https://github.com/silinternational/ecs-deploy
```

Comment:
You should probably include the MIT License text in this file from https://github.com/silinternational/ecs-deploy/blob/develop/LICENSE.